gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,16 @@
+#
+# RCTab
```

Comment:
isn't there a gradle command that generates these files? if yes, it may not make sense to put these license headers there.

Reply:
Good catch, this definitely shouldn't be appearing in here. I'll fix the auto-copyright rules and clean these up.